packages/wonder-stuff-core/src/__tests__/clone-metadata.test.ts

```diff
@@ -34,15 +33,19 @@ describe("#cloneMetadata", () => {
             boolean: "true",
             undefined: undefined,
             null: null,
-        };
+        } as const;
 
         // Act
         const result = cloneMetadata(metadata);
+        // @ts-expect-error [FEI-5011] - TS2540 - Cannot assign to 'string' because it is a read-only property.
```

Comment:
**note:** It's a shame we have all these error suppressions instead of just a judicial use of `any` on the variable from the beginning. Since these are test files, that feels like the right thing. Of course, that's manual work and would really take time, so I'm not requesting a change here. Just lamenting the tech debt/mess that is left behind.

Reply:
I expect there will be quite a bit of post-conversion work to improve our types.